feat(synthetics): Updated module import details and expanded NerdGrap… (#7041)
newrelic/docs-website: src/content/docs/apis/nerdgraph/examples/nerdgraph-synthetics-tutorial.mdx

For [simple browser monitors](#create-simple-browser):

* <var>RUNTIME_TYPE</var>: The runtime type used by your monitor. "CHROME_BROWSER" is the only accepted value.
* <var>RUNTIME_TYPE_VERSION</var>: The runtime type version used by your monitor. "100" is the only accepted value. 
* <var>SCRIPT_LANGUAGE</var>: The language used in your monitor. "JAVASCRIPT" is the only accepted value. 

For [scripted API monitors](#create-scripted-API):

* <var>SCRIPT_CONTENT</var>: The actual contents of the script. This should **not** be based64 encoded. 
* <var>RUNTIME_TYPE</var>: The runtime type used by your monitor. "NODE_API" is the only accepted value.

> * <var>RUNTIME_TYPE_VERSION</var>: The runtime type version used by your monitor. "16.10.0" is the only accepted value. 

Reviewer: ```suggestion
* <var>RUNTIME_TYPE_VERSION</var>: The runtime type used by your monitor. "16.10.0" is the only accepted value. 
```

Author: We would need to keep the words runtime type version here. I like the next suggestion you made.

> * <var>RUNTIME_TYPE_VERSION</var>: The runtime type version used by your monitor. "16.10.0" is the only accepted value. 

Reviewer: ```suggestion
* <var>RUNTIME_TYPE_VERSION</var>: The runtime type version used by your monitor. The only accepted value is `16.10.0`. 
```

Reviewer: (We try not to start sentences with numbers for formatting/style reasons. I added a codeblock around the value so users know not to include the terminal punctuation.)

Below are some sample requests to automate the creation of your synthetic monitors. 

> <Callout variant="info">

Reviewer: Also, what color do you want this? For a tip, use this code:

```
<Callout variant="tip">
text here
</Callout>
```

for important:

```
<Callout variant="important">
text here
</Callout>
```

Author: I think using a tip makes sense.

}
```
  </Collapser>

  <Collapser
    id="upgrade-monitor-runtime"
    title="Upgrade a monitor's runtime"
    >

>   Use this to upgrade a simple browser, scripted API, or scripted browser monitor from using a legacy runtime to the new Chrome 100 or Node 16.10 runtime. See [optional-fields](#optional-fields) for additional details on runtime settings. This example will upgrade the runtime for a scripted browser monitor. Use the `syntheticsUpdateSimpleBrowserMonitor` mutation for simple browser monitors and `syntheticsUpdateScriptApiMonitor` for scripted API monitors.

Reviewer: ```suggestion
You can use the below example to update a simple browser, scripted API, or scripted browser monitor to the new Chrome 100 or Node 16.10 runtimes. The following example upgrades a scripted browser monitor to the new runtime from the legacy runtime. 

* To update a simple browser monitor using the below example, change the mutation to `syntheticsUpdateSimpleBrowserMonitor` instead of `syntheticsUpdateScriptBrowserMonitor`. 
* To update a scripted API monitor using the below example, change the mutation to `syntheticsUpdateScriptApiMonitor` instead of `syntheticsUpdateScriptBrowserMonitor`.

Check out [optional-fields](#optional-fields) for additional details on runtime settings. 
```
